src/components/App/index.tsx

```diff
@@ -233,10 +235,15 @@ export default class App extends Component<Props, State> {
       loadedCounter: loadingCounter,
     };
 
-    this.setState({ images, error: '' });
+    this.setState({ images });
+  }
+
+  showError (error: Error | String) {
```

Comment:
Are we providing this function with error objects anywhere? If not, we don't need to accept it.

Also, `string` rather than `String`.

Reply:
I had thought so but I likely changed it after.